src/actions/searchActions.js

```diff
@@ -0,0 +1,24 @@
+import { instance } from '../utils/axios';
+import handleMessages from '../utils/messages';
+import actionTypes from './types';
+
+const searchActions = (filter, phrase, history) => async (dispatch) => {
+  if (!phrase) {
```

Comment:
You could consider using ternary operation here since it uses less lines of code.

Reply:
Let me try that.

Reply:
Using a ternary here became hard for me because I would have to put the `try - catch` in another function in order to use it in the ternary (which actually beats the point of having less lines of code). That function would have to be an `async` since I have an `await` in the `try`. I've tried that, but I get `Uncaught Error: Actions must be plain objects. Use custom middleware for async actions.`.
I'll just leave the `if-else` for now. I might require some help converting it into a ternary if it must be so.